app/src/pages/inside/logsPage/logsGrid/logMessageBlock/logMessageBlock.jsx

```diff
@@ -43,11 +43,13 @@ export class LogMessageBlock extends Component {
 
     return (
       <div ref={refFunction} className={cx('log-message-block')}>
-        {customProps.consoleView && <span className={cx('time')}>{dateFormat(value.time)}</span>}
         {customProps.markdownMode ? (
           <MarkdownViewer value={messageWithoutMarkdown} />
         ) : (
-          <div className={cx('log-message')}>{messageWithoutMarkdown}</div>
+          <div className={cx('log-message')}>
+            <span className={cx('time')}>{dateFormat(value.time)}</span>
```

Comment:
Please add `customProps.consoleView &&` to prevent appearing redundant time for default view (see the screenshot below).

![image](https://user-images.githubusercontent.com/31861805/77073387-421d9280-6a00-11ea-8a34-7a384ebf9c98.png)

Reply:
Fixed. Thanks!